packages/mongo/collection.js

```diff
@@ -25,6 +25,10 @@ The default id generation technique is `'STRING'`.
  * @param {Boolean} options.defineMutationMethods Set to `false` to skip setting up the mutation methods that enable insert/update/remove from client code. Default `true`.
  */
 Mongo.Collection = function Collection(name, options) {
+  if (this.constructor !== Mongo.Collection) {
```

Comment:
This should be `if (! (this instanceof Mongo.Collection))` to support subclassing.

Reply:
Updated